### PR TITLE
Sync UI submodule

### PR DESCRIPTION
**Pull request by hutaishi**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #.
- [x] Update the [CHANGES log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

sync ui: https://github.com/apache/skywalking-booster-ui/pull/59

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please update the changelog as well. Don't remove all things in the PR template, it is meaningful

**Comment by wu-sheng** [comment]
I think you copied from the old template? The latest change log is here, https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md

New component ID and new Logo changes should be landed here.

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 #### UI
 * General service instance: move `Thread Pool` from JVM to Overview, fix `JVM GC Count` calculation.
+* Add Apache ShenYu (incubating) component LOGO.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:11]: Another one is missing.

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -4,10 +4,11 @@
 
 
 #### OAP Server
-
+* Add component definition for `Apache ShenYu (incubating)`.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:7]:
```suggestion
* Add component definition(ID=127) for `Apache ShenYu (incubating)`.
```

File: docs/en/changes/changes.md (hunk 1 above)
Comment (hutaishi) [reply docs/en/changes/changes.md:11]: I've made it up. Please help me review it

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**